cvat-ui/src/components/file-manager/file-manager.tsx

```diff
@@ -156,9 +164,19 @@ export class FileManager extends React.PureComponent<Props, State> {
         }
 
         const { SHARE_MOUNT_GUIDE_URL } = config;
-        const { treeData, onUploadShareFiles, onLoadData } = this.props;
+        const {
+            treeData, sharedStorageInitialized, onUploadShareFiles, onLoadData,
+        } = this.props;
         const { expandedKeys, files } = this.state;
 
+        if (!sharedStorageInitialized) {
```

Comment:
Maybe need to move it inside and return the tab component once, what do you think?
```
<Tabs.TabPane key='share' tab='Connected file share'>
...
</Tabs.TabPane>
```

Reply:
Done